mne/channels/data/montages/standard_1005.elc

```diff
@@ -352,7 +352,7 @@ Positions
 Labels
 LPA
 RPA
-Nz
+nasion
```

Comment:
you're changing the files provided also by eeglab and fieldtrip. Can we avoid this?